challenge-082/markus-holzer/raku/ch-1.raku

```diff
@@ -0,0 +1,3 @@
+unit sub MAIN( Int $N, Int $M );
+
+say "({ join ', ', grep all( $N, $M ) %% *, 1 ..^ max $N, $M })"
```

Comment:
The "all" part is cool!

Reply:
Yeah, junctions are cool. I am using a similar code in last weeks challenge.